src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html

```diff
@@ -4,7 +4,7 @@
   </div>
   <div class="msw-student-body">
     <div class="msw-student-user-pic-ph">
-      <img src="http://via.placeholder.com/140x100">
+      <img src="/assets/img/picking.tinti.png" alt="" width="80" height="80" style="background-color:white;">
```

Comment:
Can you move it to css class?

Reply:
Already moved it, I guess I forgot to delete from the HTML.............. did it now

Reply:
Done.